install/operator/.lib.sh

```diff
@@ -149,6 +149,9 @@ FROM golang:1.13.7
 WORKDIR /go/src/${OPERATOR_GO_PACKAGE}
 ENV GO111MODULE=on
 COPY . .
+RUN go mod vendor
```

Comment:
This line is problematic & should not be necessary since the vendor directory is already resident as part of the src code directory as identified by WORKDIR (L149). It would also cause issues since it would try downloading stuff as not be able to necessarily have access to get it.

Reply:
done, thanks.